Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-                        "star": { "href": "/gists/42/star" },
-                    },
-                    "id": "42",
-                    "created_at": "2014-04-14T02:15:15Z",
-                    "description": "Description of Gist",
-                    "content": "String contents"
-                }
-
-Resource Model is an example representation of the Gist resource. It can be referenced elsewhere at any point where a response (or request) payload is expected. A resource model has *exactly* the same structure as a response payload. In this case it also includes additional description. A Resource Model may not include a response status code.
-
-<a name="refer_model"></a>
-#### Referring the Resource Model
-
-    ### Retrieve a Single Gist [GET]
-    + Response 200
-
-        [Gist][]
-
-With Gist Resource Model in place it is super easy to define an action that retrieves a single gist.
 
-> **NOTE:** The syntax for referencing a resource model is `[<resource identifier>][]`. The model must be referred as "whole" - you can't reuse just a model's body or headers.
-
-## Modifying a Resource
-Let's add an action to our Gist Resource that will allow us to modify its state and another action that deletes a Gist Resource as whole.
-
-    ### Edit a Gist [PATCH]
-    To update a Gist send a JSON with updated value for one or more of the Gist resource attributes. All attributes values (states) from the previous version of this Gist are carried over by default if not included in the hash.
-
-    + Request (application/json)
+```markdown
++ Response 200 (application/json)
 
+        [
             {
-                "content": "Updated file contents"
+                "question": "Favourite programming language?",
+                "published_at": "2014-11-11T08:40:51.620Z",
+                "url": "/questions/1",
+                "choices": [
+                    {
+                        "answer": "Swift",
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "answer": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "answer": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "answer": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
             }
+        ]
+```
+
+> **Note:** API Blueprint is **indentation sensitive**. Assets have to be **indented by four spaces** relative to it's existing indentation level. In the case of list items or sections, which are already indented. It results in having you include additional indentation, this further makes it a total of **eight spaces** when using assets inside a list item.
```

Comment:
"It results in having you include additional indentation" 

Doesn't make sense to me